Doc/c-api/arg.rst

```diff
@@ -344,6 +344,18 @@ Other objects
    in *items*.  The C arguments must correspond to the individual format units in
    *items*.  Format units for sequences may be nested.
 
+   If *items* contains format units which store
+   :ref:`borrowed buffer <c-arg-borrowed-buffer>` or
+   :term:`reference <borrowed reference>`
+   (``y``, ``y#``, ``s``, ``s#``, ``z``, ``z#``, ``S``, ``Y``, ``U``,
+   ``O`` or ``O!``) the object must be a Python tuple.
+   The *converter* for the ``O&`` format unit in *items* must not store
+   borrowed buffer or reference.
+
+   .. deprecated:: next
+      Non-tuple sequences are deprecated if *items* contains format units
+      which store borrowed buffer or reference.
```

Comment:
```suggestion
      which store borrowed buffers or references.
```

Reply:
Maybe just

```suggestion
      which store a borrowed buffer or reference.
```

Like in your previous suggestion?

Reply:
You're actually right. It's better to use the singular form because the plural form could hint that the format unit would be able to store more than one object but AFAIK, the relevant format units can only store a single borrowed buffer or a single borrowed reference (not two borrowed references with a single symbol).

Reply:
I've updated all my suggestions to singular form and left out the repetition of "borrowed" when the context is clear.